activitysim/abm/models/util/canonical_ids.py

```diff
@@ -54,6 +57,167 @@ def enumerate_tour_types(tour_flavors):
     return channels
 
 
+def read_alts_file(file_name, set_index=None):
+    try:
+        alts = simulate.read_model_alts(file_name, set_index=set_index)
+    except RuntimeError:
+        logger.warning(f"Could not find file {file_name} to determine tour flavors.")
+        return pd.DataFrame()
+    return alts
+
+
+def read_spec_file(file_name, set_index=None):
```

Comment:
It looks like `read_alts_file` and `read_spec_file` are identical.  Shall we use only one?

Reply:
Yes, only one is needed.  And this was actually caught in the flexible trip and tour id's review.  Resolved when merging in changes from develop branch.